### PR TITLE
i18n(fr) Create `image-service-reference.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Add french translation for `image-service-reference.mdx` file

### CODE REVIEW

File: src/content/docs/fr/reference/image-service-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+## Hooks
+
+### `getURL()`
+
+**Requis pour les services locaux et externes**
+
+`getURL(options: ImageTransform, imageConfig: AstroConfig['image']): string`
+
+Pour les services locaux, ce crochet renvoie l'URL du point d'accès qui génère votre image (en mode SSR et dev). Il n'est pas utilisé pendant la construction. Le point d'accès local vers lequel `getURL()` pointe peut appeler à la fois `parseURL()` et `transform()`.
```
Comment (thomasbnt) [inline src/content/docs/fr/reference/image-service-reference.mdx:166]:
```suggestion
Pour les services locaux, ce hook renvoie l'URL du point d'accès qui génère votre image (en mode SSR et dev). Il n'est pas utilisé pendant la construction. Le point d'accès local vers lequel `getURL()` pointe peut appeler à la fois `parseURL()` et `transform()`.
```

File: src/content/docs/fr/reference/image-service-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+Pour les services externes, ce crochet renvoie l'URL finale de l'image.
```
Comment (thomasbnt) [inline src/content/docs/fr/reference/image-service-reference.mdx:168]:
```suggestion
Pour les services externes, ce hook renvoie l'URL finale de l'image.
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]

File: src/content/docs/fr/reference/image-service-reference.mdx [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+Configurez le service d'image à utiliser dans `astro.config.mjs`. La configuration prend la forme suivante :
+
+```js title="astro.config.mjs"
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+  image: {
+    service: {
+      entrypoint: "your-entrypoint", // 'astro/assets/services/squoosh' | 'astro/assets/services/sharp' | string,
```
Comment (dreyfus92) [inline src/content/docs/fr/reference/image-service-reference.mdx:253]:
```suggestion
      entrypoint: "votre-point-d’entrée", // 'astro/assets/services/squoosh' | 'astro/assets/services/sharp' | string,
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]

**Comment by McFlyPartages** [comment]
@thomasbnt, @dreyfus92 vos suggestion ont été ajouté merci a vous.

LGTM

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Comment by McFlyPartages** [comment]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**